pgrx-examples/aggregate/embed.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    panic!("PGRX_EMBED is not override.");
```

Comment:
This should be, I think,
```suggestion
    panic!("PGRX_EMBED was not set.");
```